packages/octane.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Octane < Package
+  description 'The JavaScript Benchmark Suite for the modern web https://developers.google.com/octane/.'
+  homepage 'https://github.com/chromium/octane'
+  version 'v1'
+  source_url 'https://github.com/chromium/octane/archive/v1.tar.gz'
+  source_sha256 '51e9002e8391f65809c19200e44277d0e5a0adf9f7c03af46676cd321f965d68'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'nginx'
+
+  def self.install
+    system "mkdir -p #{CREW_DEST_PREFIX}/share/nginx/html/octane"
+    system "cp -r . #{CREW_DEST_PREFIX}/share/nginx/html/octane"
+  end
+
+  def self.postinstall
+    system "ps ax | grep httpd | grep -v grep | head -1 2> /dev/null > /tmp/httpd.pid"
+    if not File.zero? "/tmp/httpd.pid" then
+      puts
+      puts "Make sure httpd is not running on port 80".lightblue
+      puts "sudo apachectl stop".lightblue
```

Comment:
Shouldn't this be controlling nginx instead of apache?

Reply:
This logic only tests if Apache is using port 80 since it would create a conflict with Nginx.

Reply:
Can we check this another way? How about netstat?

Reply:
I just thought it would be easier using port 80 since it wouldn't require any extra configuration.  We could try netstat.  How would that work?